ocs_ci/deployment/deployment.py

```
@@ -3747,6 +3748,396 @@ def _patch_ramen_csv(self, csv_prefix, namespace, new_image):
+            "-o",
+            output_dirname,
+        ])
+
+        logger.info(f"[ramenctl-e2e] Running command: {e2e_cmd}")
+        logger.info(f"[ramenctl-e2e] Test results will be saved to: {output_path}")
+
+        e2e_timeout = 7200
+
+        try:
+            output = run_cmd(e2e_cmd, timeout=e2e_timeout, shell=True)
+            logger.info(f"[ramenctl-e2e] E2E tests completed successfully")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)